aform/src/components/form/ADropdown.vue

```diff
@@ -1,11 +1,10 @@
 <template>
-	<div class="autocomplete" :class="{ isOpen: isOpen }">
+	<div ref="autocomplete" class="autocomplete" :class="{ isOpen: isOpen }" v-on-click-outside="closeResultsHandler">
```

Comment:
I think the function name can be more instructive:

```suggestion
	<div ref="autocomplete" class="autocomplete" :class="{ isOpen: isOpen }" v-on-click-outside="closeDropdown">
```